tgui/packages/tgui/components/Input.js

```diff
@@ -83,6 +83,10 @@ export class Input extends Component {
     if (input) {
       input.value = toInputValue(nextValue);
     }
+
+    if (this.props.autoFocus) {
+      setTimeout(() => input.focus(), 1);
```

Comment:
This could not be done right away, I think it's not attached to the DOM yet. This works 100% of the time for me though.

Reply:
Using a timeout for this is normal, because element can only be focused on the next browser tick.